app/src/main/java/org/dhis2/usescases/programEventDetail/ProgramEventMapper.kt

```diff
@@ -32,6 +32,10 @@ class ProgramEventMapper @Inject constructor(
         val programStage =
             d2.programModule().programStages().uid(event.programStage()).blockingGet()
 
+        val dataSetOrgUnitNumber = d2.organisationUnitModule().organisationUnits()
+            .byProgramUids(listOf(event.program()!!))
+            .blockingGet().size
+
         crashReportController.addBreadCrumb(
```

Comment:
please, can you remove this breadcrumb, we no longer need it.

Reply:
breadcrumb removed